Include/README.rst

```diff
@@ -0,0 +1,63 @@
+The Python C API
+================
+
+The C API is divided into three sections:
+
+1. ``Include/``
+2. ``Include/cpython/``
+3. ``Include/internal/``
+
+
+Include: Limited API and Stable ABI
+===================================
+
+``Include/`` contains the public Limited API and the Stable ABI.  The
+Stable ABI is defined by :pep:`384`.
+
+Functions stealing references or returning borrowed references *must not*
+be added to the Limited API API or the Stable ABI.  A strong reference
+*must* be returned.  The API in ``Include/`` should not expose or leak
+implementation details.
+
+Please start a public discussion before adding new C API to ``Include/``.
+
+Functions or macros with a ``_Py`` prefix do not belong in ``Include/``.
+
+
+Include/cpython: CPython implementation details
+===============================================
+
+``Include/cpython/`` contains the public API that is excluded from the
+Limited and Stable ABI.  :pep:`384` introduced the ``Py_LIMITED_API``
+define to exclude functions from the Limited API.  When a new API is
+introduced, it has to be explicitly excluded using
+``#ifndef Py_LIMITED_API``.
+
+Functions stealing references or returning borrowed references *must not*
+be added to ``Include/cpython/``.  A strong reference *must* be
+returned.
+
+See also `bpo-35134 <https://bugs.python.org/issue35134>`_.
```

Comment:
That's an issue that contains a lot of text. What's the essence of what we can learn from there? Can't you just say what needs to be known here in this README? (And why is that issue still open?)

Reply:
I included it just as a reference. I guess we can just drop that link. I think most of what is needed from that issue is already included here.